swagger/formats.go

```diff
@@ -0,0 +1,35 @@
+package swagger
+
+import (
+	"regexp"
+
+	"github.com/go-openapi/strfmt"
+)
+
+func InitCustomFormats() {
+	m := mongoID("")
+	strfmt.Default.Add("mongo-id", &m, isMongoID)
+}
+
+var mongoRegExp = regexp.MustCompile("^[0-9a-f]{24}$")
+
+func isMongoID(str string) bool {
+	return mongoRegExp.MatchString(str)
```

Comment:
oops. seems like I am wrong there. In any case, would be nice if we used the `bson` package as the source of truth for this info. Also would be really helpful if `wag` just parsed `mongo-id` format strings into a `bson.ObjectId`.